pysatNASA/instruments/methods/omni.py

```diff
@@ -32,15 +51,15 @@ def time_shift_to_magnetic_poles(inst):
 
     # Need to fill in Vx to get an estimate of what is going on.
     inst['Vx'] = inst['Vx'].interpolate('nearest')
-    inst['Vx'] = inst['Vx'].fillna(method='backfill')
-    inst['Vx'] = inst['Vx'].fillna(method='pad')
+    inst['Vx'] = inst['Vx'].bfill()
```

Comment:
pandas updates?

Reply:
FutureWarning from pandas updates. The method kwargs are deprecated, new functions for backfill (bfill) and forward fill (ffill)